examples/cluster.html

```diff
@@ -5,19 +5,21 @@
 docs: >
   This example shows how to do clustering on point features.
 tags: "cluster, vector"
+resources:
+  - https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css
```

Comment:
Should this go in the example template instead?

Reply:
I guess I'm not clear on which examples need it.

Reply:
It is in the examples template, but when editing an example in codesandbox it needs to be loaded there too.

Reply:
Got it.  Thanks.